libpod/oci_linux.go

```diff
@@ -102,3 +103,35 @@ func (r *OCIRuntime) createContainer(ctr *Container, cgroupParent string) (err e
 
 	return err
 }
+
+func rpmVersion(path string) string {
+	output := "Unknown"
+	cmd := exec.Command("/usr/bin/rpm", "-q", "-f", path)
+	if outp, err := cmd.Output(); err == nil {
+		output = string(outp)
+	}
+	return strings.Trim(output, "\n")
+}
+
+func dpkgVersion(path string) string {
+	output := "Unknown"
+	cmd := exec.Command("/usr/bin/dpkg", "-S", path)
+	if outp, err := cmd.Output(); err == nil {
+		output = string(outp)
+	}
+	return strings.Trim(output, "\n")
+}
+
+func (r *OCIRuntime) pathPackage() string {
+	if out := rpmVersion(r.path); out != "Unknown" {
+		return out
+	}
+	return dpkgVersion(r.path)
+}
+
+func (r *OCIRuntime) conmonPackage() string {
+	if out := rpmVersion(r.conmonPath); out != "Unknown" {
+		return out
+	}
+	return dpkgVersion(r.conmonPath)
+}
```

Comment:
Any consideration to putting this in a pkg .. something like distro ?

Reply:
We can always move this around in a later PR, if someone thinks it is worth while elsewhere.